package.json

```diff
@@ -41,14 +41,11 @@
   },
   "pnpm": {
     "patchedDependencies": {
-      "@docusaurus/theme-search-algolia@2.3.1": "patches/@docusaurus__theme-search-algolia@2.3.1.patch"
+      "@docusaurus/theme-search-algolia@3.0.0-alpha.0": "patches/@docusaurus__theme-search-algolia@2.3.1.patch"
     },
     "peerDependencyRules": {
       "ignoreMissing": [
-        "@docusaurus/core",
-        "@docusaurus/mdx-loader",
-        "@docusaurus/plugin-content-blog",
-        "@docusaurus/preset-classic"
+        "react-json-view"
```

Comment:
We still need this peer dependency override until https://github.com/facebook/docusaurus/pull/9116 is released